src/components/Wizard/SpotlightWizard/SpotlightActionInput.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { FieldPathByValue, useFormContext } from "react-hook-form";
+
+import ErrorMessage from "../../ErrorMessage";
+import FormRow from "../FormRow";
+import SpotlightWizardFormData, { SpotlightActionFormData } from "./formData";
+import {
+  RegisteredFormCheck,
+  RegisteredFormControl,
+} from "../../RegisteredFormControl";
+import { validateJsonAsObject } from "../validators";
+
+interface SpotlightActionInputProps {
+  controlPrefix: FieldPathByValue<
+    SpotlightWizardFormData,
+    SpotlightActionFormData
+  >;
+  required?: boolean;
+  disabled?: boolean;
+}
+
+export default function SpotlightActionInput({
+  controlPrefix,
+  required = false,
+  disabled = false,
+}: SpotlightActionInputProps) {
+  const { register } = useFormContext<SpotlightWizardFormData>();
+
+  return (
+    <FormRow label="Action">
+      <FormRow
+        label="Navigate?"
+        containerClassName="form-input-check"
+        helpText="Advance to the next screen when this action triggers"
+      >
+        <RegisteredFormCheck
+          name={`${controlPrefix}.navigate`}
+          register={register}
+          disabled={disabled}
+        />
+      </FormRow>
+      <FormRow label="Type">
```

Comment:
Might be nice to have some helptext describing action type, or point to some documentation